src/components/popovers/TradeSettingsPopover.vue

```diff
@@ -24,10 +24,7 @@
       </div>
       <AppSlippageForm class="mt-1" />
     </div>
-    <div
-      v-if="appTradeInterface === TradeInterface.BALANCER && !hideLiquidity"
-      class="mt-6"
-    >
+    <div v-if="!hideLiquidity" class="mt-6">
```

Comment:
this is just re-enabling previously disabled behaviour... it should be possible to select liquidity source when using the new trade interface (since it integrates Balancer too)